hazelcast/src/main/java/com/hazelcast/client/console/HazelcastCommandLine.java

```diff
@@ -84,6 +84,9 @@
 import static java.util.Collections.emptyList;
 import static java.util.concurrent.TimeUnit.SECONDS;
 
+/**
+ * Picocli implementation of the {@code hz-cli} command
+ */
```

Comment:
Also `com.hazelcast.client` shouldn't be covered, this specific Javadoc doesn't add much useful information...

And if this really is public API, we should not make the classes below non-public.